department-of-veterans-affairs/caseflow
# Fix Pagination Button import in story

### Description
Fix import statement so that storybook renders

### Acceptance Criteria
- [ ] Storybook renders story

### Testing Plan
1. Start storybook `make storybook`
2. Go to Components/PaginationButton
3. Verify story renders

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1207" alt="Screen Shot 2021-03-11 at 6 04 03 PM" src="https://user-images.githubusercontent.com/68879427/110881172-c366d880-8294-11eb-8416-5c500c8c0eda.png">|<img width="1210" alt="Screen Shot 2021-03-11 at 6 10 39 PM" src="https://user-images.githubusercontent.com/68879427/110881383-20628e80-8295-11eb-9831-6b375be33ad9.png">
